### DISCUSSION TITLE
Overdue membership dues

### DISCUSSION THREAD

**Discussion by ebullient** [commonhaus/foundation#46] [Q&A]
c/o @trustin :

> Should we also mention what happens when a member does not pay the due? e.g. not paying the due leads to suspension and not paying for 3+ months leads to revocation?

  https://github.com/commonhaus/foundation-draft/commit/5a33a7c3676a0fa09317ea8ce269ecb0b52bf8df#r135054481

**Comment by kenfinnigan** [comment]
@trustin would you be willing to provide a small PR with your suggested change covering non-payment of fees to https://github.com/commonhaus/foundation-draft/blob/main/bylaws/2-cf-membership.md#general-members
